README.md

```diff
@@ -1,11 +1,9 @@
 ## MQTT Agent and Demo Tasks (including OTA) - Using coreMQTT
 
-Note:  The MQTT agent and associated example project are functional but not yet complete. Be aware the agent does not yet comply with our code quality standards and is not yet fully tested. There is a chance its APIs will change before its official first release.  Feedback, suggestions and comments are welcome on the [FreeRTOS support forum](https://forums.freertos.org).
+This repository provides demonstration examples of using the [coreMQTT Agent library](https://github.com/FreeRTOS/coreMQTT-Agent) with FreeRTOS. The coreMQTT Agent library is an extension on top of [coreMQTT](https://github.com/FreeRTOS/coreMQTT) that provides MQTT APIs with thread safety. The examples here share a single MQTT connection amongst multiple concurrent tasks, without requiring the management of any synchronization primitives from the application.
```

Comment:
I'd start with the most important points about the demo. For ex: "the demo uses OTA, Device Shadow, and Device Defender in multiple threads."

Reply:
Reworded